### DISCUSSION TITLE
[A-L] (20 Settembre 2024) Esercizio 3

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#419] [Esami A-L]
Sia $\Large R \subseteq \mathbb{N}$ una relazione binaria con la seguente proprietà:

<div align="center">

per ogni $\Large n,m,q \in \mathbb{N}$, se $\Large (n,m) \in R$ e $\Large (n,q) \in R$ allora $\Large (m,q) \notin R$.

</div>

1. R può essere transitiva? [1 punto]

2. R può essere riflessiva? [2 punti]

3. Per ogni $\Large n \in \mathbb{N}$ esistono infiniti $\Large m \in \mathbb{N}$ tali che $\Large (n,m) \in R$? [2 punti]

**Comment by cek753** [comment]
1 Vero, solo se vuota perché così non ci sono coppie che violano la transitivitá
2 Falso
3 Vero
